.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Pull request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  tests:
+    name: Test
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
```

Comment:
I know there aren't any tags/releases of this yet, but in the future it would be nice to have some so that we don't float on `main` here.

Reply:
we are planning on changing this in the future once we are more stable.